File: PooCC01CNetz/Semana03/Python/Ejercicios1/Escuela.py
from Alumno import *
class Escuela:
    alu = Alumno()
    alu.setNombre("Juan")
    alu.setMatricula(12345)
    print("Nombre: " + str(alu.getNombre()))
    print("Matricula: " + str(alu.getMatricula()))

File: PooCC01CNetz/Semana03/Java/Ejercicios1/Escuela.java
public class Escuela {
    public static void main(String[] args) {
        Alumno alu = new Alumno();
        alu.setNombre("Juan");
        alu.setMatricula(12345);
        System.out.println("Nombre: " + alu.getNombre());
        System.out.println("Matricula: " + alu.getMatricula());
    }
}